openmetadata-spec/src/main/resources/json/schema/security/credentials/gcpCredentials.json

```diff
@@ -9,7 +9,20 @@
     "gcpCredentialsPath": {
       "title": "GCP Credentials Path",
       "description": "Pass the path of file containing the GCP credentials info",
-      "type": "string"
+      "type": "object",
+      "properties": {
+        "path": {
```

Comment:
as we have modified the json schema, we will require migration to move gcpCredentialsPath which was previously a string to object so gcpCredentialsPath -> gcpCredentialsPath.path
You can test the migrations by creating Bigquery service in older version and then upgrade to this PR, start the server, and it should fail until the migrations are applied

Reply:
Where should I write migration ? otherwise I can move the param outside. Need a bit of guidance here 🙏